contrib/survey_cake/udfs/survey_cake_csv_uploader.py

```diff
@@ -20,6 +21,7 @@ def __init__(self, year: int, filename: str):
         self.dimension_table_filepath = (
             self.filename.parent / f"{self.filename.stem}_dimension.csv"
         )
+        self.existing_question_and_id_dict = self._get_existing_question_and_id()
```

Comment:
need to get `existing` question id first so that we can use the existing id for `2021` 's fact table